.simplecov

```diff
@@ -1,8 +1,15 @@
 # frozen_string_literal: true
+
 SimpleCov.start do
-  add_filter %r{^/spec/}
+  load_profile "test_frameworks"
```

Comment:
It seems this does something with features directory as well. My main concern there is I want to make sure its actually tracking the step files.

Reply:
> My main concern there is I want to make sure its actually tracking the step files.

Confirmed. this excludes test, specs, features

Ref: https://github.com/simplecov-ruby/simplecov/blob/main/lib/simplecov/profiles/test_frameworks.rb

I understand this use case, I will suggest a different approach